src/mobility/components/BenefitTile.tsx

```diff
@@ -0,0 +1,149 @@
+import {ThemeText, screenReaderPause} from '@atb/components/text';
+import {TileWithButton} from '@atb/components/tile';
+import {StyleSheet} from '@atb/theme';
+import {InteractiveColor} from '@atb/theme/colors';
+import {View, ViewStyle} from 'react-native';
+import {BenefitImageAsset} from './BenefitImage';
+import {FareProductBenefitType} from '../use-operator-benefits-for-fare-product';
+import {getTextForLanguage, useTranslation} from '@atb/translations';
+import {MobilityTexts} from '@atb/translations/screens/subscreens/MobilityTexts';
+import {Map} from '@atb/assets/svg/mono-icons/map';
+import {
+  MapFilterType,
+  MobilityMapFilterType,
+  useUserMapFilters,
+} from '@atb/components/map';
+import {FormFactor} from '@atb/api/types/generated/mobility-types_v2';
+import {getNewFilterState} from '../utils';
+import {useOperators} from '../use-operators';
+
+type BenefitCardProps = {
+  interactiveColor: InteractiveColor;
```

Comment:
Is this parameter used?

Reply:
Not more than once. Would you prefer if I didn't expose it?